docs/meshstack.billing.md

```diff
@@ -239,6 +239,24 @@ The following configuration options are available at `mesh.kraken.productcatalog
 
 Each project in meshStack is associated with a Chargeback Account. meshStack periodically generates [chargeback statements](meshcloud.project-metering.md#chargeback-statements).
 
+The attributes that shall be part of the billing info on the chargeback statements can be configured as relevant meta keys at `mesh.kraken.api.statements`
```

Comment:
f: This does not look like it's sourced from a proper snippet like 
`<!--snippet:mesh.kraken.api.statements-->`

did you write it yourself @sttomm ?

Reply:
yeah, you are right. i manually added it. I now added it as a real snippet